models/user.py

```diff
@@ -1,13 +1,40 @@
 from bson import ObjectId
+
 from models.base_model import BaseModel
 
 
 class User(BaseModel):
     def __init__(self, collection='users'):
         super().__init__(collection)
 
-    def all(self):
-        return self.client.db.users.find({})
+    def all(self, validated):
+        match = {}
+        if validated == 'true':
+            match.update({'validated': True})
+        elif validated == 'false':
+            match.update({'validated': False})
+
+        users = self.client.db.users.aggregate([
+            {
+                '$match': match
+            },
+            {
+                '$project': {
```

Comment:
Extraer proyección a un diccionario propio

Reply:
Extract to "_private" method